src/components/types/types.tsx

```diff
@@ -1,11 +1,16 @@
 export interface ILessonBack {
   author_id: number;
   category_id: number;
+  created_at: string;
   description: string;
   id: number;
+  image_link?: string;
```

Comment:
Please, not required fields (with question mark) write at the end, in first place required:

author_id: ...
category_id: ...
....
image_link?: ...

Reply:
done